### ISSUE TITLE
Mobile Column is Hardcoded in Making the Notifiable Model

### ISSUE THREAD

**Issue by MajdTaweel** [mohammad-fouladgar/laravel-otp#9]
Hello @mohammad-fouladgar,

I noticed something that didn't add up after changing my **otp** config, specifically after changing the mobile column:
`TypeError : App\Models\SomeNotifiableModel::getMobileForOTPNotification(): Return value must be of type string, null returned`

This is caused by the mobile column being hardcoded in making the notifiable model:

```
class OTPBroker
{
    .
    .
    .

    private function makeNotifiable(string $mobile): OTPNotifiable
    {
        return $this->userRepository->getModel()->make(['mobile' => $mobile]);
    }
}

```

**Suggested Fix**
```
-        return $this->userRepository->getModel()->make(['mobile' => $mobile]);
+        $mobileColumn = config('otp.mobile_column');
+        return $this->userRepository->getModel()->make([$mobileColumn => $mobile]);

```

Regards,
Majd

**Comment by mohammad-fouladgar** [comment]
@MajdTaweel Thank you for reporting. Feel free to submit a PR to fix this issue.

**Comment by mohammad-fouladgar** [comment]
This issue is closed by  PR #10.
